content/workers/platform/routing/service-bindings.md

```diff
@@ -0,0 +1,94 @@
+
+A Service binding allows you to send HTTP requests to another Worker without those requests going over the Internet. The request immediately invokes the downstream Worker, reducing latency as compared to a request to a third-party service. You can invoke other Workers directly from your code; making it possible to communicate with shared services managed by other teams with differing test and release processes. Those services do not need to be hooked up to publicly accessible endpoints; as such Service bindings facilitate private services to communicate with one another.
+
+Service bindings allow you to:
+* Segment multiple use cases into separate Workers Services that can be explicitly invoked from your code.
+* Achieve better composability on the Workers platform using service-oriented architecture.
+* Create private microservices, to be conditionally invoked from other edge-facing services.
+
+
+### Interface
+Service bindings use the standard [Fetch](../../runtime-apis/fetch) API. You can continue to use your existing utilities and libraries - a Workers Service binding will trigger a [FetchEvent](../../runtime-apis/fetch-event). While the interface among Service bindings is HTTP, the networking is not. Unlike the typical microservice architecture, where services communicate over a network and can suffer from latency or interruption, Service bindings are a zero-cost abstraction. When one Worker invokes another, there is no network delay and the request is executed immediately.
+
+![Service bindings are a zero-cost abstraction](../media/../routing/media/service-bindings-comparison.png)
+
+### Shared Resources
+Workers connected to one another via Service bindings share the CPU resources of the top-level request. A single thread is allocated and reused amongst these Workers. This means no idle resources are wasted while work is performed across various Workers.
+
+### Setting a Service binding
+You can manage Workers Service bindings in [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Settings**> **Variables** > **Service bindings** > **Edit variables**. You can also change the environment of a Workers Service binding, so you can target a specific version of a Workers Service.
+
+![To configure a Service binding, head to your Worker's Settings tab](../media/service-bindings-config.png)
+
+Workers bound to your Worker will be listed in the [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Triggers** tab, under **Bound Services**. Your team can easily view cross-service dependencies in this manner.
+
+![Your team can view cross-service dependencies in the Triggers tab](../media/service-bindings-triggers.png)
+
+## Composing an example Worker
+
+### Authentication Workers Service
+
+Following authentication Workers Service code responds with `200` in case `x-custom-token` request matches `SECRET_TOKEN` secret binding.
```

Comment:
"The following"? This sentence needs revision, it's very abrupt and has no 'what is the developer doing, how is the developer doing it, why is the developer doing it' structure.